test/videojs-http-streaming.test.js

```diff
@@ -3094,8 +3094,7 @@ QUnit.test('passes useCueTags hls option to master playlist controller', functio
   videojs.options.hls = origHlsOptions;
 });
 
-// TODO: This test fails intermittently. Turn on when fixed to always pass.
-QUnit.skip('populates quality levels list when available', function(assert) {
+QUnit.test('populates quality levels list when available', function(assert) {
```

Comment:
This test was previously skipped because of intermittent failures. I think fixing this bug fixes the test.